tez/tez.sh

```diff
@@ -132,14 +135,18 @@ function configure_master_node() {
   systemctl restart hadoop-yarn-timelineserver
   systemctl status hadoop-yarn-timelineserver  # Ensure it started successfully
 
-  # Restart hive server2
-  systemctl restart hive-server2
-  systemctl status hive-server2  # Ensure it started successfully
+  if [[ "${HOSTNAME}" == "${master_hostname}" ]]; then
```

Comment:
In case all masters run a hive-server2 in the future, I'd rather restart hive-server2 if it's installed/enabled in systemd, rather than only restarting on -m-0..

Reply:
Sure. It may be possible that all masters will run hive-server2 in future. I could use `systemctl show --property=LoadState` to check if it's installed or not. The result of that command could be "loaded" or "not-found".